formats/placeholder/publisher/details.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
```

Comment:
Generally, we disallow undefined keys for good usability reasons and it would be a shame to lose that. For Whitehall case studies, we are doing something very similar by sending the item with a format of "case_study", but a rendering_app of "whitehall-frontend", rather than "government-frontend". Maybe we can defer what you suggest until we need it?